ddtrace/internal/telemetry/writer.py

```diff
@@ -220,13 +221,35 @@ def __init__(self):
         self._namespace = MetricNamespace()
         self._logs = []  # type: List[Dict[str, Any]]
 
+    def _fork_writer(self):
+        # type: () -> None
+        self.disable()
+        self.reset_queues()
+
+    def enable(self):
+        # type: () -> bool
+        """
+        Enable the instrumentation telemetry collection service. If the service has already been
+        activated before, this method does nothing. Use ``disable`` to turn off the telemetry collection service.
+        """
+        if config._telemetry_metrics_enabled:
+            if self.status == ServiceStatus.RUNNING:
+                return True
+
+            self._enabled = True
+            self.start()
+
+            atexit.register(self.stop)
+            return True
+        return False
```

Comment:
```suggestion
            return super(TelemetryLogsMetricsWriter, self).enable()
```

Reply:
Yes this is an enhancement/fix that I tried to sneak into your PR 😅. If DD_INSTRUMENTATION_TELEMETRY_ENABLED is False we should never submit telemetry.